finetuner-workflow/finetune-workflow.yaml

```diff
@@ -573,18 +573,16 @@ spec:
                           values:
                             - "{{workflow.parameters.region}}"
             containers:
-              - name: kfserving-container
+              - name: http-inference-container
                 image: "{{workflow.parameters.finetuner_image}}:{{workflow.parameters.finetuner_tag}}"
                 imagePullPolicy: IfNotPresent
                 command: ["/usr/bin/bash", "-c"]
-                args: ["/usr/bin/python3 /app/inference.py"]
+                args: ["/usr/bin/python3 /app/inference.py --model=${INFERENCE_MODEL} --port=${INFERENCE_PORT}"]
                 env:
                   - name: INFERENCE_PORT
                     value: "80"
                   - name: INFERENCE_MODEL
                     value: /mnt/pvc/{{ inputs.parameters.model_path }}
-                  - name: STORAGE_URI # KFServing mounts the PVC at /mnt/pvc/
```

Comment:
This env var needs to be kept as it allows for mounting the PVC so we can load models from it.